src/main/java/com/ariatemplates/attester/maven/ArtifactExtractor.java

```diff
@@ -88,12 +96,18 @@ public static String getPropertyName(Dependency dependency) {
     }
 
     public static File getOutputDirectory(Artifact artifact) {
```

Comment:
Just small remark for the code change, I notice this static method is no more used and thus you can remove it.